lib/asm-docs/generated/asm-docs-riscv64.ts

```diff
@@ -0,0 +1,3809 @@
+                "tooltip": "ANDI, ORI, XORI are logical operations that perform bitwise AND, OR, and XOR on register rs1 and the sign-extended 12-bit immediate and place the result in rd",
+                "url": "https://five-embeddev.com/riscv-isa-manual/latest/rv32.html#integer-register-immediate-instructions"
+            };
+
+        case "ADD":
+            return {
+                "html": "<div>\n<span class=\"opcode\">rd, rs1, rs2</span>\n\n<div>add t0, t1, t2 slti t3, t2, 0 slt t4, t0, t1 bne t3, t4, overflow In RV64I, checks of 32-bit signed additions can be optimized further by comparing the results of ADD and ADDW on the operands.</div>\n</div>",
+                "tooltip": "add t0, t1, t2 slti t3, t2, 0 slt t4, t0, t1 bne t3, t4, overflow In RV64I, checks of 32-bit signed additions can be optimized further by comparing the results of ADD and ADDW on the operands.",
+                "url": "https://five-embeddev.com/riscv-isa-manual/latest/rv32.html#integer-computational-instructions"
+            };
+
+        case "SUB":
+            return {
+                "html": "<div>\n<span class=\"opcode\">rd, rs1, rs2</span>\n\n<div>SUB performs the subtraction of rs2 from rs1</div>\n</div>",
+                "tooltip": "SUB performs the subtraction of rs2 from rs1",
+                "url": "https://five-embeddev.com/riscv-isa-manual/latest/rv32.html#integer-register-register-operations"
+            };
+
+        case "SLL":
+            return {
+                "html": "<div>\n<span class=\"opcode\">rd, rs1, rs2</span>\n\n<div>SLL, SRL, and SRA perform logical left, logical right, and arithmetic right shifts on the value in register rs1 by the shift amount held in the lower 5 bits of register rs2 .</div>\n</div>",
+                "tooltip": "SLL, SRL, and SRA perform logical left, logical right, and arithmetic right shifts on the value in register rs1 by the shift amount held in the lower 5 bits of register rs2 .",
+                "url": "https://five-embeddev.com/riscv-isa-manual/latest/rv32.html#integer-register-register-operations"
+            };
+
+        case "SLT":
+            return {
+                "html": "<div>\n<span class=\"opcode\">rd, rs1, rs2</span>\n\n<div>SLT and SLTU perform signed and unsigned compares respectively, writing 1 to rd if</div>\n</div>",
+                "tooltip": "SLT and SLTU perform signed and unsigned compares respectively, writing 1 to rd if",
+                "url": "https://five-embeddev.com/riscv-isa-manual/latest/rv32.html#integer-register-register-operations"
+            };
+
+        case "SLTU":
+            return {
+                "html": "<div>\n<span class=\"opcode\">rd, rs1, rs2</span>\n\n<div>Note, SLTU rd , x0 , rs2 sets rd to 1 if rs2 is not equal to zero, otherwise sets rd to zero (assembler pseudoinstruction SNEZ rd, rs )</div>\n</div>",
```

Comment:
This seems like it must be cut out of a larger paragraph that probably has more context. It doesn't really describe what SLTU does (which is an unsigned integer comparison).